books/free-programming-books-langs.md

```diff
@@ -1223,6 +1224,11 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Fastify - Latest Documentation](https://www.fastify.io/docs/latest) (HTML)
 
 
+#### Hydrogen
+
+* [Build a Hydrogen storefront](https://shopify.dev/custom-storefronts/hydrogen/getting-started/tutorial) (HTML)
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. I would put it in subjects/
Misc, unless you can find other books on application frameworks, commerce or something else.

Reply:
added another tutorial

Reply:
that's more of a blog post. We''ll add the category and maybe reorganize later